Library/Homebrew/cmd/update.sh

```diff
@@ -393,8 +393,15 @@ EOS
       odie "Git must be installed and in your PATH!"
     fi
   fi
-  export GIT_TERMINAL_PROMPT="0"
-  export GIT_SSH_COMMAND="ssh -oBatchMode=yes"
+
+  if [[ -z "$GIT_TERMINAL_PROMPT" ]]
```

Comment:
Instead of `if/then` you can do
```bash
export GIT_TERMINAL_PROMPT=${GIT_TERMINAL_PROMPT:-"0"}
export GIT_SSH_COMMAND=${GIT_SSH_COMMAND:-"ssh -oBatchMode=yes"}